daemon/stats_collector_unix.go

```diff
@@ -35,6 +36,11 @@ func (daemon *Daemon) newStatsCollector(interval time.Duration) *statsCollector
 		clockTicksPerSecond: uint64(system.GetClockTicks()),
 		bufReader:           bufio.NewReaderSize(nil, 128),
 	}
+	meminfo, err := sysinfo.ReadMemInfo()
```

Comment:
Can we store this on the daemon like we do some other sysinfo related things instead of calling it every time we request stats?